src/graphql/models/AvatarTypeEnum.js

```diff
@@ -0,0 +1,9 @@
+import { GraphQLEnumType } from 'graphql';
+import { AvatarTypes } from 'util/user';
+
+export default new GraphQLEnumType({
+  name: 'sEnum',
```

Comment:
Looks like a typo lol
Let's align the name with the file name.

```suggestion
  name: 'AvatarTypeEnum',
```